Functions/Quantification/xASL_quant_ASL.m

```diff
@@ -94,6 +94,8 @@
 
 if ~x.modules.asl.ApplyQuantification(3)
     fprintf('%s\n','We skip the scaling of a.u. to label intensity');
+	% In case we don't run quantification, we still need to average the PWI image
+	PWI = xASL_im_ASLSubtractionAveraging(x, [], PWI4D);
```

Comment:
Can't we not move this line outside of this if else statement, because it should be independent of ApplyQuantification(3)? Now we repeat ourselves.

Reply:
We can't really do that. Previously. We operated with only PWI in this function.
Now this function takes in PWI4D and then turns it to PWI for single-PLD quantification and leaves it at PWI4D for BASIL. 

And for quantification OFF, it doesn't do anything. But unfortunately, we still need PWI for that case. It was previously available, now it's not. So we can't do it elsewhere and we don't repeat anything. In other branches of this IF, something else happends.